objects.proto

```diff
@@ -66,3 +66,7 @@ message Decision {
   State source = 1;
   Edge edge = 2;
 }
+
+message Path {
+  repeated string edge_ids = 1;
+}
```

Comment:
Edge `edge_ids` er vel ikke unikke for komponenter? Så hvordan skal GUIen vide hvilken `Path` tilhøre hvilket komponent?

Reply:
Så vi resultere en liste af path, en path for over individual component. 
Så hvis vi har en reachability query fx. `reachability: (HalfAdm1 && HalfAdm2) -> [L12, L14](y<3, x>8); [L13, L15](y<2, x>5)`, så vil den først path vil være for `HalfAdm1` og den anden path er for `HalfAdm2`.

Reply:
Vi laver edge IDs unikke på GUIen så det bør ikke være et problem

Reply:
Okay det giver menig

Reply:
> Vi laver edge IDs unikke på GUIen så det bør ikke være et problem

Vel ikke på kryds af komponenter?

Reply:
Joh tænkte at lave det til GUIDs. Ødelægger det noget i backend?

Reply:
Nej, de gør det langt nemmere at implementere.